### PR TITLE
Updates banner code

### PR DESCRIPTION
**Pull request by anselmbradford**
Updates banner code to better match generator-cf. Part of DF-1857.

## Changes

- Updates grunt-banner to 0.4.0.
- Updates ascii art to match that in generator-cf yeoman generator.
- Updates bower.json to match generator-cf in respect to banner code by making `authors` array a single `author` field and adding an `email` field and value.
- Regenerates JS assets.

## Testing

- Run `grunt jsdev` to see new banner code in common.js file.

## Review

- @sebworks 
- @jimmynotjim 
- @KimberlyMunoz 

## Preview

screen shot 2015-05-14 at 10 18 28 am

## Notes

- Banner is not being inserted into CSS (this is the existing behavior), because it is trying to be applied to `*.min.css`. This will be corrected when the CSS moves to a /dist/ directory.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -192,21 +192,21 @@ module.exports = function(grunt) {
...
+      ' *   `"Ybd$"′   $$     $$`"YbdP"′   $$`"YbdP"′\n' +
+      ' *                     $$\n' +
+      ' *                     $$\n' +
+      ' *                     $$\n' +
       ' *\n' +
       ' *  <%= pkg.name %> - v<%= pkg.version %>\n' +
-      ' *  <%= pkg.homepage %>' +
-      ' *  A public domain work of the Consumer Financial Protection Bureau\n' +
+      ' *  <%= pkg.homepage %>\n' +
+      ' *  Licensed <%= pkg.license %> by <%= pkg.author.name %> <<%= pkg.author.email %>>\n' +
```
Comment (Scotchester) [inline Gruntfile.js]: This is going to output "Licensed https://github.com/cfpb/cfgov-refresh/blob/master/TERMS.md" if you don't update the value on `bower.json`.

I know this line comes from generator-cf, but I say leave it as-is for now. I'm not satisfied with the output of this for our projects using the default answers. The phrase "Licensed CC0 by ..." has a number of legal and grammatical issues.

I have more thoughts on this that don't really belong in this comment.

File: bower.json [hunk 2]
```diff
@@ -7,12 +7,12 @@
...
-      "name": "Consumer Financial Protection Bureau",
-      "url": "https://cfpb.github.io/"
-    }
-  ],
+  "author":
+  {
+    "name": "Consumer Financial Protection Bureau",
+    "email": "tech@cfpb.gov",
+    "url": "https://cfpb.github.io/"
+  },
```
Comment (Scotchester) [inline bower.json]: Can't do this, according to the [spec](https://github.com/bower/bower.json-spec#authors).

Comment (anselmbradford) [inline bower.json]: Good catch, I was going off generator-cf. Opened https://github.com/cfpb/generator-cf/issues/71

File: Gruntfile.js (hunk 1 above)
Comment (anselmbradford) [inline Gruntfile.js]: Reverted in https://github.com/cfpb/cfgov-refresh/commit/cd4543d6c4da636c8f562be5876cb7699caf84e2

File: bower.json (hunk 2 above)
Comment (anselmbradford) [inline bower.json]: Fixed in https://github.com/cfpb/cfgov-refresh/commit/cd4543d6c4da636c8f562be5876cb7699caf84e2 Also updated `url` > `homepage` per the spec.

File: CHANGELOG.md [hunk 3]
```diff
@@ -24,6 +24,9 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 - Updated grunt-eslint to `^13.0.0`.
 - Moved eslint config file to home directory.
 - Moved jQuery download to package.json.
+- Updated grunt-banner to `^0.4.0` and updates banner ascii art and copy.
+- Changed bower.json authors array field to single field author,
```
Comment (Scotchester) [inline CHANGELOG.md]: This is no longer totally accurate.

Comment (anselmbradford) [inline CHANGELOG.md]: Nice! Good catch. Fixed in https://github.com/cfpb/cfgov-refresh/commit/eb1a0ce70b80154fbc714bfb03284efbfd381937 and https://github.com/cfpb/cfgov-refresh/commit/109ca0963f1934bba2b97f549cc9044af089a6d3

Comment (anselmbradford) [inline CHANGELOG.md]: Also fixed in https://github.com/cfpb/cfgov-refresh/commit/8dbd55c246b84f0cc5e76a416224a19dba55e815

Comment (Scotchester) [inline CHANGELOG.md]: Well, that's a good candidate for squashing ;)

**Comment by Scotchester** [comment]
All good! :+1:

**Comment by anselmbradford** [comment]
'ppreciate it!!

**Pull request merged**